syntax = "proto3";
package cloudfirewall;

service Heartbeat {
  rpc Ping(PingRequest) returns (PingResponse) {}
}

message PingRequest {
  string node_id = 1;
  string node_name = 2;
}

message PingResponse {
  string extra_info = 1;
}
